redbaron/base_nodes.py

```diff
@@ -1020,6 +1021,14 @@ def replace(self, new_node):
         self.__class__ = new_node.__class__  # YOLO
         self.__init__(new_node.fst(), parent=self.parent, on_attribute=self.on_attribute)
 
+    def rename(self, new_value):
+        if self.type in ('def', 'class'):
+            setattr(self, 'name', new_value)
```

Comment:
In first time I thought about this solution, but this attribute is outside of __init__ method. 
So, I think it is not correct way. Please fix me if I am wrong.